app/code/Magento/CatalogUrlRewrite/Test/Unit/Observer/ProductProcessUrlRewriteSavingObserverTest.php

```diff
@@ -205,13 +205,6 @@ public function testExecuteUrlKey(
             ->method('getIsChangedCategories')
             ->will($this->returnValue($isChangedCategories));
 
-        $this->urlPersist->expects($this->exactly($expectedDeleteCount))->method('deleteByData')->with([
```

Comment:
I believe we should remove `$expectedDeleteCount` variable (parameter) from the function as well since it's unused.

Reply:
hi @rogyar 
Thanks for your review, I really appreciate it! 👍 
Backporting tests always give me headaches 😄 I hope they run smoothly through travis now

Reply:
Please, fix the mentioned issue via a new commit. After that the tests will be executed once again.